core/src/main/java/org/mule/api/context/notification/ProcessorsTrace.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.api.context.notification;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Keeps context information about the message processors that were executed as part of the processing of an event.
+ * 
+ * This will only be enabled it the log is set to debug or if the system property {@code mule.flowTrace} is set to {@code true}. If not enabled, the list will always be empty.
```

Comment:
Not sure that this information goes here, because not only affects the executed processors but also the traces